### ISSUE TITLE
data inconsistency between pretraining model data and proteinnet

### ISSUE THREAD

**Issue by zzhang-QSI** [aqlaboratory/rgn#21]
Hi, alquraishi 

I tried to train the model using CASP12 tfrecord data download from proteinnet: https://sharehost.hms.harvard.edu/sysbio/alquraishi/proteinnet/tfrecords/casp12.tar.gz
I found the training_90 data is different from pretrained model (CASP12) 
RGN12/runs/CASP12/ProteinNet12Thinning90 
https://sharehost.hms.harvard.edu/sysbio/alquraishi/rgn_models/RGN12.tar.gz

may I know which data is better?  
Using proteinnet data , I found `numEvoEntries` change to 21 instead of 42, and the training loss always nan.

Looking forward to hearing from you.
Thank you
